Assets/Scripts/Actor Components/Buff.cs

```diff
@@ -42,6 +75,8 @@ private void RPC_ApplyBuff()
         combat.AttackEffectLayer = buffedTargetLayer;
         spriteRenderer.color = buffedColor;
         IsBuffed = true;
+
+        StartTimer();
```

Comment:
If I'm not mistaken, by putting `StartTimer` in `RPC_ApplyBuff`, the timer will run on all clients. This then results in `RemoveBuff` being called on both clients, which in turn results in both clients sending RPCs with `RPC_RemoveBuff`. It might be better to place `StartTimer` in `ApplyBuff` and `StopTimer` in `RemoveBuff` so that there are less RPC calls.